gitsetup/pom.xml

```diff
@@ -23,7 +23,6 @@
         <version>2.14.0-SNAPSHOT</version>
     </parent>
 
-    <groupId>ddf</groupId>
```

Comment:
❓ Why remove the group id? It is clearer if we keep it. This one might actually cause a problem when running the githooks.